### PR TITLE
Output requested node resources

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java [hunk 1]
```diff
@@ -212,6 +214,14 @@ private void toSlime(History history, Cursor array) {
         }
     }
 
+    private void toSlime(NodeResources resources, Cursor object) {
+        object.setDouble("vcpu", resources.vcpu());
+        object.setDouble("memoryGb", resources.memoryGb());
+        object.setDouble("diskGb", resources.diskGb());
+        object.setDouble("bandwithGgps", resources.bandwidthGbps());
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java]: bandwithGgps -> bandwidthGbps

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java [hunk 2]
```diff
@@ -212,6 +214,14 @@ private void toSlime(History history, Cursor array) {
...
+        object.setString("disk-speed", serializer.toString(resources.diskSpeed()));
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java]: Stick to camel case for key names?

**Comment by bratseth** [comment]
Thanks, you are right on both counts of course

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java [hunk 3]
```diff
@@ -212,6 +214,14 @@ private void toSlime(History history, Cursor array) {
         }
     }
 
+    private void toSlime(NodeResources resources, Cursor object) {
+        object.setDouble("vcpu", resources.vcpu());
+        object.setDouble("memoryGb", resources.memoryGb());
+        object.setDouble("diskGb", resources.diskGb());
+        object.setDouble("bandwithGbps", resources.bandwidthGbps());
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java]: There's still a typo here.. :laughing:

Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java]: You want absolutely no typos in the entire word? High bar :-(

**Review by freva — APPROVED** [review]

**Pull request merged**